lib/LedgerSMB/Admin/Command/user.pm

```diff
@@ -112,20 +115,20 @@ sub _option_spec {
             # employee
             'dob=s' => \$dob,
             'employeenumber=s' => \$employeenumber,
-            'end_date=s' => \$end_date,
-            'is_manager!' => \$is_manager,
+            'end-date=s' => \$end_date,
+            'is-manager!' => \$is_manager,
             'manager=s' => \$manager,
             'role=s' => \$role,
             'sales!' => \$is_sales,
```

Comment:
This is rather inconsistent: declaring a sales works with `--sales` but declaring a manager is `--is-manager` ?!

Reply:
Fixed